examples/dcgan/dcgan.gop

```diff
@@ -28,13 +30,108 @@ var (
 	cuda       = flag.Bool("cuda", false, "enables cuda")
 	dryRun     = flag.Bool("dry-run", false, "check a single training cycle works")
 	ngpu       = flag.Int("ngpu", 1, "number of GPUs to use")
-	netG       = flag.String("netG", "", "path to netG (to continue training)")
-	netD       = flag.String("netD", "", "path to netD (to continue training)")
+	netGPath   = flag.String("pathG", "", "path to netG (to continue training)")
+	netDPath   = flag.String("pathD", "", "path to netD (to continue training)")
 	outf       = flag.String("outf", ".", "folder to output images and model checkpoints")
 	manualSeed = flag.Int("manualSeed", 0, "manual seed")
 	classes    = flag.String("classes", "bedroom", "comma separated list of classes for the lsun data set")
 )
 
+
+type Generator struct {
+	ngpu int
+	main torch.Module
```

Comment:
Let us don't name the field name. Instead, let us keep it unnamed.

Reply:
The `main` data member follows the Python codes. And it will also be used here:

```go
func (g* Generator) Forward(x torch.Tensor) output torch.Tensor {
	if x.IsCuda && g.ngpu > 1 {
		output = torch.DataParallel(g.main, x, range(g.ngpu))
	} else {
		output = g.main.Forward(x)
	}
}
```

It has to access the `main` field.